src/Framework/Exceptions/Primitives/DuplicateStripeAccountName.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Give\Framework\Exceptions\Primitives;
+
+/**
+ * Class DuplicateStripeAccountName
+ * @package GiveStripe\Infrastructure\Exceptions
+ *
+ * @unreleased
+ */
+class DuplicateStripeAccountName extends Exception {
```

Comment:
This is not the right place to put these (this and the other two exceptions). The primitives are just that, primitive exceptions native to the PHP language — nothing more. All additional exceptions should go with the domain they belong to.

Reply:
@JasonTheAdams Thank you suggestion. Core refactored: https://github.com/impress-org/givewp/compare/5528061fc6ca...07b57469366b